run-automation-test.md

```diff
@@ -0,0 +1,169 @@
+### 3.2 Configure automation test script
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference. 
+> Note: In this guideline, we will use the Node.js sample (samples/javascript folder) as an example. In sample/javascript folder, we will use `android-app-test` script. You can choose `android-app-test.js` if you want to run tests on Android or `ios-app-test.js` if on iOS.
+
+Use Kobiton’s desired capabilities settings to configure your test scripts. Our settings include all the required configurations you need to automate your test using the Appium Framework and to run your automation test on real devices in the Kobiton cloud.
+
+Open automation test script file in your repository or create a new one.
+
+Replace `desiredCaps` in the script with the ones collected in the previous step. Then,replace value of `app` element in `desiredCaps` with the `APP_URL` environment variable (The value of `app` element is the download URL of your app. If your automation test script is in other languages, please use the proper method to access this environment variable).
+
+Example in Node.js:
+
+```javascript
+var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                process.env.APP_URL, 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<DEVICE_NAME>',
+  platformVersion:    '<DEVICE_VERSION>',
+  platformName:       '<DEVICE_PLATFORM_NAME>' 
+}
+```
+### 3.3 Configure TeamCity project
+**Attaching environment variable to Teamcity**
++ In your project in TeamCity, on the left side of the page, click **Parameters**. Then click **Add new parameter** button. 
+
+![parameters](./assets/param.png)
+
++ Add environment variables. For the **Kind** category, please choose **Environment variable (env.)**. 
+
+![env.username](./assets/param_name.png)
+
+Here are the variables we need. For the sake of this guideline, please set the name for these variables like below.
+
+FILE_FORMAT: Your app file extension. `apk` for Android, `ipa` for iOS.
+
+APP_ID: Your app ID
+
+HOCKEYAPP_TOKEN: Hockey App API token
+
+KOBITON_API_KEY: Your Kobiton API Key
+
+KOBITON_USERNAME: Your Kobiton username
+
+![environment variables](./assets/2-env.png)
+
+**Setup TeamCity automation testing**
+
+1. In your TeamCity project, on the left side of the page, click **Build Steps** -> **Add build step** to set build steps.
+
+![build step](./assets/build_step.png)
+
+2. On the right side of the page, for the **Runner type** category, choose **Command Line**.
+3. Add below commands in **Custom script** section.
+
+```
+  url=$(node getAppUrl)
+  npm install
+  APP_URL=$url npm run <YOUR_AUTOMATION_SCRIPT_EXECUTION_COMMAND>
+```
+>Note: url=$(node getAppUrl): this command is required to get the app download URL and set value of url.
```

Comment:
Explain `getAppUrl` is the `path to the node application`.
Reminder: This is how a directory tree in a react native project look like:
```
root
----- react native
----- testscript
```
We aren't 100% sure that the test script is in the root folder of the repo, so tell them to access the file by `path`, not just `name`